public/scss/_main.scss

```diff
@@ -0,0 +1,14 @@
+@import "./abstracts/_variables.scss";
+@import "./abstracts/_mixins.scss";
+@import "./base/_index.scss";
+@import "./components/_app.scss";
+@import "./components/_slider.scss";
+@import "./layouts/_footer.scss";
+@import "./layouts/_header.scss";
+@import "./pages/_home.scss";
+@import "./pages/_login.scss";
+body {
+  padding: 0;
+  margin: 0;
```

Comment:
This is the main sass file. It shouldn't begin with an underscore. Just rename it back to `main.scss`.